Plan/src/main/java/com/djrapitops/plan/utilities/metrics/BStatsVelocity.java

```diff
@@ -0,0 +1,41 @@
+package com.djrapitops.plan.utilities.metrics;
+
+import com.djrapitops.plan.PlanVelocity;
+import com.djrapitops.plan.system.database.databases.Database;
+import com.djrapitops.plan.system.info.connection.ConnectionSystem;
+import com.djrapitops.plugin.api.utility.log.Log;
+// bstats does not currently support velocity
+import org.bstats.velocity.Metrics;
+
+import java.io.Serializable;
+
+public class BStatsVelocity {
```

Comment:
I think the velocity metrics classes should be removed until bStats comes up with actual implementation - It will be trivial to implement them afterwards.